recipes/logic1/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+context:
+  version: 0.1.0
+
+package:
+  name: logic1
+  version: ${{ version }}
+
+source:
+  git: https://github.com/logic1-eu/logic1.git
+  tag: v${{ version }}
+
+build:
+  number: 0
+  noarch: python
```

Comment:
Sorry, but I don't follow. I understand that I should make a change with regards how minimum/maximum Python versions are enforced. Probably I should remove the constraints from `python >=3.11,<3.12` in lines 19 and 24, and use a different mechanism.

The example you reference applies to `meta.yaml`, but not to `recipe.yaml`. I found this hint, though: https://github.com/conda-forge/staged-recipes/blob/14f3a9c46a6abe964d3887025f4bec477ac716f4/recipes/example-v1/recipe.yaml#L37-L46

And based on that I would do:

```suggestion
  noarch: python
  skip:
    - match(python, "<3.11")
    - match(python, ">3.11")
```

Please provide a little more guidance so that I can make the correct change. Thanks!

Reply:
I think what you want to do is add `python_min` to the `context`, and then in 
`host`: `- python {{ python_min }}`
`run`: `python >={{ python_min }},<3.12`

For `recipe.yaml`, the `test` `requirements` don't quite line up since you can't specify it for the `python` section, but what I've done elsewhere is the following:
https://github.com/conda-forge/jsonpickle-feedstock/blob/main/recipe/recipe.yaml#L34-L38

So it repeats the import, but checks that it works with the `python_min` via `python -c "import ..."`.

Hope that helps.

Reply:
Alright, I think that's fixed now.